libraries/AP_HAL_ChibiOS/hwdef/CubeSolo/hwdef.dat

```diff
@@ -46,3 +46,41 @@ COMPASS AK09916:probe_ICM20948 0 ROTATION_ROLL_180_YAW_90
 
 # also probe for external compasses
 define HAL_PROBE_EXTERNAL_I2C_COMPASSES
+
+DMA_NOSHARE USART6_TX USART6_RX USART2_TX ADC1
+
+# telem2 does not need DMA
+undef USART3_TX
+undef USART3_RX
+undef PD8
+undef PD9
+PD8 USART3_TX USART3 NODMA
+PD9 USART3_RX USART3 NODMA
+
+# GPS does not need DMA
+undef PA0
+undef PA1
+PA0 UART4_TX UART4 NODMA
+PA1 UART4_RX UART4 NODMA
+
+# serial5 does not need DMA
```

Comment:
serial5 is bared as part of the external expansion port.   It might very well need DMA.

Reply:
but is its dma more important than main sensors? without DMA UART still works